x-pack/plugins/observability/public/pages/slos/components/slo_list_item.tsx

```diff
@@ -146,13 +143,18 @@ export function SloListItem({
               <EuiFlexGroup direction="column" gutterSize="m">
                 <EuiFlexItem>
                   <EuiText size="s">
-                    <EuiLink data-test-subj="o11ySloListItemLink" onClick={handleViewDetails}>
-                      {slo.name}
-                    </EuiLink>
+                    {slo.summary ? (
```

Comment:
If we're optimistically updating when creating a new item, `summary` doesn't exist yet. In that case we need to block users from navigating to the page, as the optimistic update has a client side generated UUID which doesn't exist.